src/conditional-flow/string-conditions.js

```diff
@@ -48,7 +59,15 @@ function getMiddleLetter (val1) {
 // Autumn - September to November
 // Winter - December to February
 function seasonForMonth (monthName) {
-
+if (monthName==="March"||monthName==="April"||monthName==="May"){
```

Comment:
This works well! As I mentioned above, it might be a little bit more readable if you used arrays containing the seasons and to check them like that. Both are good implementations though!